src/current/_includes/releases/v22.2/v22.2.15.md

```diff
@@ -0,0 +1,75 @@
+- The `cockroach debug pebble` command now works correctly with [encrypted stores](https://www.cockroachlabs.com/docs/v22.2/security-reference/encryption#cockroachdb-self-hosted-clusters) which do not use the default `cockroach-data` path without having to also pass `--store`. [#110509][#110509]
+- When upgrading from previous releases, it was possible that ambiguous references to [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) would fail to be resolved by ID if they were in the same schema. Additionally, the logging is improved for ambiguous cases where the reference cannot be resolved. [#110721][#110721]
+- Fixed a bug where [`RESET `](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#reset-a-storage-parameter-to-its-default-value)[(`ttl_expire_after`)](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expire_after) could incorrectly remove [`ttl_expiration_expression`](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expiration_expression). [#110750][#110750]
+- Fixed a bug where [`format_type`](https://www.cockroachlabs.com/docs/v22.2/functions-and-operators#compatibility-functions) builtin did not honor `typemod` information for [array](https://www.cockroachlabs.com/docs/v22.2/array) types, leading to incorrect output. [#110939][#110939]
+- External connection URLs now accept the scheme `azure-blob` for connections to [Azure](https://www.cockroachlabs.com/docs/v22.2/use-cloud-storage#url-format) Blob Storage and the scheme `azure-kms` for connections to Azure KMS. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage.
+- Fixed a bug where `indoption` inside [`pg_index`](https://www.cockroachlabs.com/docs/v22.2/pg-catalog#data-exposed-by-pg_catalog) was not properly encoded, causing it to be unable to be decoded as `int2vector`. [#111956][#111956]
+- Fixed a bug where the [optimizer](https://www.cockroachlabs.com/docs/v22.2/cost-based-optimizer) failed to honor the `statement_timeout` session setting when generating constrained index scans for queries with large `IN` lists or `= ANY` predicates on multiple index key columns, which may lead to an [out-of-memory crash (OOM)](https://www.cockroachlabs.com/docs/v22.2/cluster-setup-troubleshooting#out-of-memory-oom-crash) on the node. [#112075][#112075]
+- Fixed a bug that caused internal errors during query [optimization](https://www.cockroachlabs.com/docs/v22.2/cost-based-optimizer) in rare cases. The bug has been present since version 2.1.11, but it is more likely to occur in version 21.2.0 and later, though it is still rare. The bug only presents when a query contains `min` and `max` aggregate functions. [#112299][#112299]
```

Comment:
Suggest making 'aggregate functions' a link to https://www.cockroachlabs.com/docs/v22.2/functions-and-operators#aggregate-functions

Reply:
added link